patches/blockprotocol+0.0.1.patch

```diff
@@ -41,7 +41,64 @@ index 205630a..1cf433a 100644
    sourceEntityId: string;
    destinationEntityId: string;
    destinationEntityVersionId?: string | null;
-@@ -170,8 +171,7 @@ export type BlockProtocolCreateLinksFunction = {
+@@ -153,6 +154,42 @@ export type BlockProtocolLinkGroup = {
+   links: BlockProtocolLink[];
+ };
+ 
++export type BlockProtocolAggregateOperation = {
++  entityTypeId?: string;
++  entityTypeVersionId?: string | null;
++  multiFilter?: {
++    filters: {
++      field: string;
++      value: string;
++      operator: BlockProtocolFilterOperatorType;
++    }[]
++    operator: BlockProtocolMultiFilterOperatorType;
++  } | null;
++  multiSort?: {
++    field: string;
++    desc?: boolean | null;
++  }[] | null;
++  itemsPerPage?: number;
++  pageNumber?: number;
++  pageCount?: number;
++};
++
++export type BlockProtocolLinkedAggregationUpdated = {
++  sourceAccountId: string;
++  sourceEntityId: string;
++  path: string;
++  operation: BlockProtocolAggregateOperation;
++}
++
++export type BlockProtocolLinkedAggregation = {
++  sourceAccountId: string;
++  sourceEntityId: string;
++  path: string;
++  operation: BlockProtocolAggregateOperation;
++  results: Array<BlockProtocolEntity>;
++}
```

Comment:
Repetition here you could get rid of - e.g. have a single `BlockLinkedAggregation` interface and then another type that adds results

Reply:
These types should now be updated